src/test/java/com/orzechazo/eshop/bootstrap/tests/BootstrapUsersAndOrders.java

```diff
@@ -0,0 +1,108 @@
+package com.orzechazo.eshop.bootstrap.tests;
+
+import com.orzechazo.eshop.domain.Order;
+import com.orzechazo.eshop.domain.User;
+import com.orzechazo.eshop.repositories.OrderRepository;
+import com.orzechazo.eshop.repositories.UserRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
+public class BootstrapUsersAndOrders {
+    private final OrderRepository orderRepository;
+    private final UserRepository userRepository;
+
+    public BootstrapUsersAndOrders(OrderRepository orderRepository, UserRepository userRepository) {
+        this.orderRepository = orderRepository;
+        this.userRepository = userRepository;
+    }
+    private static final LocalDateTime DATE_TIME = LocalDateTime.of(2024,1,1,0,0);
+    public static final String DB_ORDER_ID1 = "ORDER1";
+    public static final String DB_ORDER_ID2 = "ORDER2";
+    public static final String DB_ORDER_ID3 = "ORDER3";
+    public static final String DB_ORDER_ID4 = "ORDER4";
+    public static final String DB_ORDER_ID5 = "ORDER5";
+    public static final String DB_USER_LOGIN = "user1";
+    private List<Order> orders;
+    private List<Order> user1_orders;
+    private List<User> users = new ArrayList<>();
+
+    public void loadData() {
+
+        log.debug("Loading Orders...");
+
+        Order order1 = new Order();
+        order1.setOrderId(DB_ORDER_ID1);
+        order1.setOrderDate(DATE_TIME);
+        order1.setAdmissionDate(DATE_TIME);
+        order1.setRealizationDate(DATE_TIME);
+        order1.setPaymentDate(DATE_TIME);
+        order1.setTotalPrice(new BigDecimal("100"));
```

Comment:
This should be the place where you create ORDER1_TOTAL_PRICE, because here you are creating order1 and setting price, not in OrderServiceImplIT class